src/sidekick/de.css

```
@@ -158,6 +158,10 @@
   content: "Vorschau-Generierung fehlgeschlagen. Bitte versuche es später noch einmal.";
 }
 
+.hlx-sk-overlay .modal.modal-preview-not-gdoc::before {
+  content: "Vorschau fehlgeschlagen. Muss ein Google-Dokument oder eine Tabelle sein.";
```

I would keep this consistent with the translation on L158 above (or change both):
```suggestion
  content: "Vorschau-Generierung fehlgeschlagen. Muss ein Google-Dokument oder eine Tabelle sein.";
```

---

I used google translate. This is not one of my languages. :)